Clinical-Genomics/loqusdbapi
# Unfreeze loqusdb and pymongo-related dependencies

### This PR adds | fixes:
- Fix #36 

**How to prepare for test**:
- [x] run: `docker-compose up`

### How to test:
- Make sure images compile as expected and that the api is available: http://127.0.0.1:9000/docs
- Run` curl -X GET "http://127.0.0.1:9000/variants/1_880086_T_C"`

### Expected outcome:
- [x] The API should return the stats

### Review:

This [version](https://semver.org/) is a:
